rackhd_stub.go

```diff
@@ -0,0 +1,365 @@
+package main
```

Comment:
this whole file isn't readable at all, try to make it a bit more readable. code used for testing should still be readable